.github/workflows/pull-request.yml

```
@@ -253,6 +253,35 @@ jobs:
+          name: Check if Talawa Admin app starts in Docker
+          runs-on: ubuntu-latest
+          steps:
+             - name: Checkout the Repository
+             uses: actions/checkout@v4
+        
+             - name: Set up Docker
+             run: |
+                  sudo apt-get update
+                  sudo apt-get install -y docker.io
+        
+              - name: Run Docker Container
```

yamllint

[warning] wrong indentation: expected 15 but found 14

(indentation)